background.js

```diff
@@ -1,3 +1,19 @@
 chrome.runtime.onInstalled.addListener(() => {
-
+    
 });
+
+//lists items that should be added to list when popup opens
+let itemsToAdd = [];
+
+//listener for highlight and popup open
+chrome.runtime.onMessage.addListener(function(request, sender, sendResponse){
+    if (request.text == "popup opened") {
+        console.log("sent retrievable items");
+        chrome.runtime.sendMessage({retrieve: "retrieve", itemsToAdd: itemsToAdd});
+        itemsToAdd = [];
```

Comment:
Do we really need to set `itemsToAdd = []` again if we've already set it equal to `[]` above?